RealmSwift/SwiftUI.swift

```diff
@@ -158,16 +161,21 @@ private class ObservableStorage<ObservedType>: ObservableObject where ObservedTy
         willSet {
             if newValue != value {
                 objectWillChange.send()
-                self.objectWillChange = ObservableStoragePublisher(newValue)
+                self.objectWillChange = ObservableStoragePublisher(newValue, keyPaths)
             }
         }
     }
 
     var objectWillChange: ObservableStoragePublisher<ObservedType>
-
-    init(_ value: ObservedType) {
+    var keyPaths: [String]?
+//    init(_ value: ObservedType) {
```

Comment:
Can be removed?

Reply:
Resolved in [b8e9ee0](https://github.com/realm/realm-cocoa/pull/7249/commits/b8e9ee02afed35f06f77bd688886ad6b7aea627d)